### ISSUE TITLE
Multiple empty lines

### ISSUE THREAD

**Issue by Najdan** [realm/SwiftLint#305]
It should show warning if there are multiple empty lines one after another. Didn't noticed this rule and it would be good to have it.

Example:
```swift
func a() {
}





func b() {
}



var abc = "test"
```
Should be something like this:
```swift
func a() {
}

func b() {
}

var abc = "test"
```

**Comment by jpsim** [comment]
Closing as a duplicate of #548.
